test.sh

```diff
@@ -134,8 +134,8 @@ function verify() {
 function expected_errors() {
   local instance_file="$1"
   local type
-  type=$(yq -r '.properties | type' "$instance_file")
-  if [ "$type" != "object" ]; then
+  type=$(yq -r '.properties | type' "$instance_file" | sed 's/!!//')
```

Comment:
What does the `sed` command do?

Reply:
I was getting YAML type tags `!!map` instead of YAML type `map` or JSON type `object`. I couldn't figure out why you were getting different behaviour. 

Locally installed yq version on Mac:
```
$ brew install yq
$ yq --version
yq (https://github.com/mikefarah/yq/) version v4.40.5
```

Reply:
Resolved in the comment above